subgraph/src/unlock.ts

```diff
@@ -59,9 +59,9 @@ export function handleNewLock(event: NewLock): void {
   // fetch lock version
   let lockContract = PublicLockMerged.bind(lockAddress)
   let version = BigInt.fromI32(0)
-  let publicLockVersion = lockContract.try_publicLockVersion()
+  let publicLockVersion = lockContract.try_publicLockVersion1()
```

Comment:
Can you explain this change please?

Reply:
Because I reverted back to generating the ABIs instead of having them added back into the repo, there are multiple versions of this view function added to the ABI. The one before caused the sync to fail as there is not a function that returns a bigInt, only a i32. The naming is generated by codegen.